docusaurus/docs-internals/editor-controls/multivalue-controls/gallery.md

```diff
@@ -162,43 +225,110 @@ In this example, the user cannot remove the last item in the gallery.
 }
 ```
 
-### Return value
+#### Default value examples
 
-Returns a stringified array of objects with the following values:
+In this example, setting a default value for the gallery control.
 
 ```js
 {
-  id: number;
-  uid: string;
-  fileName: string;
-  width: number;
-  height: number;
+  id: "gallery",
+  type: "gallery",
+  default: {
+    value: [
+      {
+        id: "1",
+        uid:"dbe6479df22c4ef2ac6553aa241d2c5c.jpg",
+        fileName: "image1.png",
+        width: 800,
+        height: 600
+      },
+      {
+        id: "2",
+        uid:"fbe6479df22c4ef2ac6555te241d2c5c.jpg",
+        fileName: "image2.jpg",
+        width: 1024,
+        height: 768
+      }
+    ]
+  }
```

Comment:
Will this work ? Because later will be parsed
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/9ca65ba9-f865-4301-838d-f9d182b6b4eb)